src/app/commands/cmd_options.cpp

```diff
@@ -1766,6 +1770,20 @@ class OptionsWindow : public app::gen::Options {
     }
   }
 
+  // Function to determine if the input extension is the default theme
+  static bool isExtensionADuplicatedDefaultTheme(const Extension* e) {
+    if (!e->isDefaultTheme())
+      return false;
+    auto userThemePaths =
+      themeFolders({"extensions", skin::SkinTheme::kThemesFolderName});
+    for (auto& p : userThemePaths) {
+      // Has the user path (p) the same path of the extension (e->path())?
+      if (std::strncmp(e->path().c_str(), p.c_str(), p.size()) == 0)
```

Comment:
Since e->path() and p are std::strings, this could be replaced by:
```c++
      if (e->path() == p)
```

Reply:
Unfortunately that expression does not work, nor does the following one:
```cpp
if (e->path().compare(p) == 0)
```
I'll return with
```cpp
if (std::strncmp(e->path().c_str(), p.c_str(), p.size()) == 0)
```

Reply:
Sorry, you are right, I didn't see that you where comparing up to p.size() characters and not the full strings.

In this case, the right one would be:
```c++
if ( e->path().compare(0, p.size(), p) == 0)
```
But I don't think it makes any difference...so I'm fine if you leave the std::strncmp() call as is.